### PR TITLE
Submitting Assignment 2 - Adam Saylor

### PR DESCRIPTION
**Pull request by sayloradm**

### CODE REVIEW

**Comment by sleddog** [comment]
Missing java code newAlphabet.java that is referenced in the README.md.  Please commit this file to this branch.

**Comment by sleddog** [comment]
much better, minor nit - remove this swap file `problems/kattis/A-New-Alphabet/java/sayloradm/.README.md.swp` feel free to add this pattern to the top level .gitignore file

File: problems/kattis/A-New-Alphabet/java/sayloradm/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+Read me for assignment 2 -
+
+------- Solution Explanation --------
```
Comment (dzumpano) [inline problems/kattis/A-New-Alphabet/java/sayloradm/README.md:3]: May have wanted to include the original problem for clarity

Comment (dzumpano) [inline problems/kattis/A-New-Alphabet/java/sayloradm/README.md:3]: May have wanted to include original problem for clarity

File: problems/kattis/A-New-Alphabet/java/sayloradm/newAlphabet.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+				newLetter += i;
+				break;
+			
+			}
+		}
+		
+		return newLetter;
+	}
+
+}
```
Comment (dzumpano) [inline problems/kattis/A-New-Alphabet/java/sayloradm/newAlphabet.java:157]: Code is well written and easy to read. Not too much to say

**Review by dzumpano — COMMENTED** [review]
Looks good

**Review by sleddog — APPROVED** [review]

**Review by blakestanger — APPROVED** [review]
+10 
CapAdam
NICE WORK

**Pull request merged**